<ui version="4.0" >
 <class>CSServerConfig</class>
 <widget class="QWidget" name="CSServerConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Server range IP</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="horizontalSpacing" >
       <number>0</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Inicial IP address: </string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="initialValue_4" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Initial IP</string>
        </property>
        <property name="frame" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly" >
         <bool>false</bool>
        </property>
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="initialValue_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Initial IP</string>
        </property>
        <property name="frame" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly" >
         <bool>false</bool>
        </property>
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="initialValue_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Initial IP</string>
        </property>
        <property name="frame" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly" >
         <bool>false</bool>
        </property>
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QSpinBox" name="initialValue" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Initial IP</string>
        </property>
        <property name="frame" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly" >
         <bool>false</bool>
        </property>
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Final IP address:   </string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>1</string>
        </property>
        <property name="scaledContents" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>1</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>1</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QSpinBox" name="finalValue" >
        <property name="mouseTracking" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Final IP</string>
        </property>
        <property name="frame" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly" >
         <bool>false</bool>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Ping options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Time internal ping (min.): </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="interval" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label</zorder>
     <zorder>interval</zorder>
     <zorder>horizontalSpacer_2</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>initialValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>finalValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>289</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>289</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>initialValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSServerConfig</receiver>
   <slot>publishInitalIP(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>289</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>226</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finalValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSServerConfig</receiver>
   <slot>publishFinalIP(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>289</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel" >
     <x>226</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interval</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSServerConfig</receiver>
   <slot>publishPingInterval(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>290</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel" >
     <x>226</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CSServerConfig</receiver>
   <slot>configEnds()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>300</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel" >
     <x>310</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>initialValue_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel" >
     <x>124</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>initialValue_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>165</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel" >
     <x>170</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>initialValue_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>220</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>configEnds()</slot>
  <slot>publishInitalIP(int)</slot>
  <slot>publishFinalIP(int)</slot>
  <slot>publishPingInterval(int)</slot>
 </slots>
</ui>
